applications/StructuralMechanicsApplication/structural_mechanics_application_variables.h

```diff
@@ -38,7 +38,7 @@ namespace Kratos
 
     enum class SofteningType {Linear = 0, Exponential = 1, HardeningDamage = 2};
 
-    enum class TangentOperatorEstimation {Analytic = 0, FirstOrderPerturbation = 1, SecondOrderPerturbation = 2, Secant = 3};
+    enum class TangentOperatorEstimation {Analytic = 0, FirstOrderPerturbation = 1, SecondOrderPerturbation = 2, Secant = 3, SecondOrderPerturbationV2 = 4};
```

Comment:
Ok, is not 4th order, is SecondOrderPerturbationV2. In what consists?, can you add a comment so we are not confused?

(did you add the test?)

Reply:
I'll add a test, indeed